api/src/main/java/org/pmiops/workbench/api/CohortReviewController.java

```diff
@@ -289,6 +294,11 @@
         org.pmiops.workbench.model.CohortReview responseReview = TO_CLIENT_COHORTREVIEW.apply(cohortReview);
```

Comment:
Also, we could change this Function to take a CohortReview and a PageRequest like so:
TO_CLIENT_COHORTREVIEW.apply(cohortReview, PageRequest);
 That would eliminate lines 297-300

Reply:
As discussed offline, we won't do this because we need `TO_CLIENT_COHORTREVIEW` to maintain compatible with `.streams().map()`